render-wasm/src/shapes/fills.rs

```diff
@@ -80,17 +116,39 @@ impl ImageFill {
 pub enum Fill {
     Solid(Color),
     LinearGradient(Gradient),
+    RadialGradient(Gradient),
     Image(ImageFill),
 }
 
 impl Fill {
-    pub fn new_linear_gradient(start: (f32, f32), end: (f32, f32), opacity: f32) -> Self {
+    pub fn new_linear_gradient(
+        start: (f32, f32),
+        end: (f32, f32),
+        opacity: f32,
+        width: f32,
```

Comment:
If we're not using `width` in the linear gradient, let's remove it from the params list and set it to zero in the new struct.

Reply:
Thank you very much! fixed! :)